src/gui/transferlistmodel.cpp

```diff
@@ -59,6 +59,25 @@ static bool isDarkTheme();
 
 TransferListModel::TransferListModel(QObject *parent)
     : QAbstractListModel(parent)
+    , m_stateForegroundColors {{BitTorrent::TorrentState::Unknown, getColorByState(BitTorrent::TorrentState::Unknown)},
```

Comment:
Looks a bit weird to me, how about this:
```c++
, m_stateForegroundColors {
    {BitTorrent::TorrentState::Unknown, getColorByState(BitTorrent::TorrentState::Unknown)},
    // ...
}
```

Reply:
done